app/lib/window.ts

```diff
@@ -94,12 +94,19 @@ export class Window {
         } else {
             if (process.platform === 'darwin') {
                 bwOptions.titleBarStyle = 'hidden'
+                // If not macOS and native appearance is not toggled, use WCO.
+            } else {
```

Comment:
Does WCO work on Linux? This code path implies it does, the comment in [titleBar.component.scss](https://github.com/Eugeny/tabby/pull/8707/files#diff-d148ef5900e1ff0d7deaf652d073de91b29f7bf9c814006c71fda13d72acc853) implies it doesn't.

Reply:
[It doesn't](https://www.electronjs.org/docs/latest/tutorial/window-customization#window-controls-overlay-macos-windows), but good catch, will fix